source/StabilizerModel.cpp

```diff
@@ -7,7 +7,7 @@ constexpr unsigned int SUCCESS_CUTOFF = 60;
 
 StabilizerModel::StabilizerModel()
 	: currState(StabilizerModel::StabilizerState::Inactive), progress(0), endTime(0) {
-	srand(time(nullptr));
+	srand((unsigned int)time(nullptr));
```

Comment:
Does this need to be changed to `uint8_t`? Not sure if it's passed around through network synchronization or not.

Reply:
Nah, that's just used as a seed for the local RNG.